tools/devtool

```diff
@@ -81,6 +87,10 @@ MY_NAME="Firecracker $(basename "$0")"
 # Full path to the Firecracker tools dir on the host.
 FC_TOOLS_DIR=$(cd "$(dirname "$0")" && pwd)
 
+# Relative path to the Firecracker tools dir on the host.
+REL_FC_TOOLS_DIR=$(dirname "$0")
+MODULE_PY_DIR=`echo $REL_FC_TOOLS_DIR | sed 's/\//\./g'`
```

Comment:
This method of calling Python code seems a bit hacky and adds obscurity to the code.

Instead of using it like this, we could still use plain bash, without needing Python3 on the host, and add multiple entry points in the container. For example, to wrap around say, we can add a file in the container with the following content:

```
# Assuming that this file is located in tools/ctr_utils/say
import tools.shield_utils as su

import sys

if __name__ == "__main__":
    # Pass the argv list to the target function
    su.say(*sys.argv)
```

And the bash entry point would just look like this:
```
say() {
    run_devctr -- tools/ctr_utils/say $1
}
```

If we need to handle return codes from the launched application, we should be able to pick those up.

Reply:
This might work. An issue I see with this approach is that we may need to access host resources from within the docker container (filesystem and devices), under the R/W permissions. For the shield case, we would need to expose sysfs cgroups, parse some files and modify them. I will experiment with a similar setup and see if we can get rid of python3 host dependency.